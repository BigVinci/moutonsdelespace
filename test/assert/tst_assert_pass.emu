#-------
#TEST_RETURN_CODE=PASS           
#TEST_COMMENT="Teste fonctionnement standard"
#-------

load elf/boucle.o
assert reg $24 -4
assert reg $24 4 
assert byte 0x1080 48
assert byte 0x7051777 -4
assert word 0x7fff7 4
assert word 0xcd77777 -4
