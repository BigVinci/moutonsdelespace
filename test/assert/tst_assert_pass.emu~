#-------
#TEST_RETURN_CODE=PASS           
#TEST_COMMENT="Teste fonctionnement standard"
#-------

load elf/boucle.o
assert reg $24 0
assert byte 0x1000 32
assert word 0x1004 20512
