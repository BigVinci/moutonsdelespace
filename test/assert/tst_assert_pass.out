
------ Fichier ELF "elf/boucle.o" : sections lues lors du chargement ------
Virtual memory map (7 segments)
.text   	r-x	Vaddr: 0x00001000	Size: 24 bytes
.data   	rw-	Vaddr: 0x00002000	Size: 4 bytes
.bss    	rw-	Vaddr: 0x00003000	Size: 0 bytes
[heap]  	r--	Vaddr: 0x00005000	Size: 4286545920 bytes
[lib]   	r--	Vaddr: 0xff7fd000	Size: 8192 bytes
[stack] 	rw-	Vaddr: 0xff7ff000	Size: 8388608 bytes
[vsyscall]	r-x	Vaddr: 0xfffff000	Size: 4095 bytes
Symbol table has 7 syms:
Symbol            .text (0    bytes @ 0x00000000): section
Symbol            .data (0    bytes @ 0x00000000): section
Symbol             .bss (0    bytes @ 0x00000000): section
Symbol         .reginfo (0    bytes @ 0x00000000): section
Symbol             .pdr (0    bytes @ 0x00000000): section
Symbol           _start (0    bytes @ 0x00000000): object
Symbol                X (0    bytes @ 0x00000000): no type

Les deux valeurs sont identiques. 
Les deux valeurs sont identiques. 
